packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -2438,6 +2445,28 @@ curl -v -X DELETE \
 ##### Response Example
 
 
+```http
+`204 No Content`
+```
+
+#### Delete user asynchronously
+
+
+##### Request example
+
+
+```bash
+curl -v -X DELETE \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-H "Prefer: respond-async" \
+"https://${yourOktaDomain}/api/v1/users/00ub0oNGTSWTBKOLGLNR?sendEmail=true"
```

Comment:
Is the style to have a dummy ID or `{id}` placeholder here instead of a real user ID?

Reply:
Looking at a few other examples on this page, it looks like using a real user ID is standard.